classes/Hook.php

```diff
@@ -683,6 +683,8 @@ public static function exec($hook_name, $hook_args = array(), $id_module = null,
                     $display = Hook::coreCallHook($moduleInstance, 'hook'.$hook_name, $hook_args);
                 } elseif ($hook_retro_callable) {
                     $display = Hook::coreCallHook($moduleInstance, 'hook'.$retro_hook_name, $hook_args);
+                } else {
+                    continue;
```

Comment:
Can you explain this `else` ? The test is already made [line 674](https://github.com/PrestaShop/PrestaShop/pull/6267/files#diff-e38418d265bff448f525a7844d93f350R674)